.github/workflows/go.yml

```diff
@@ -23,6 +23,10 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
 
+    - name: govulncheck
+      run: |
+        go run golang.org/x/vuln/cmd/govulncheck@latest
```

Comment:
Are there ever published vulnerabilities in their database that do not have fixes yet? If yes, that could block devs from making this check pass. 

An alternative could be to put this into its own job which is a required merge check. (I have no idea any checks are required today in this repo.)

Reply:
There is a potential for that. This is something that I was thinking about yesterday. There is an open issue to add exclusions to be ignored. I had a look at implementing that and I may do that in the coming weeks. I'll leave this for now.